ui/components/configuratorComponents/patternConfigurator.js

```diff
@@ -45,6 +45,12 @@ const useStyles = makeStyles((theme) => ({
       height : '100%',
     }
   },
+  patterntype : {
+    padding : '0px',
+    paddingBottom : '5px' ,
+    paddingTop : '5px',
```

Comment:
Do these need to be statically sized or can this be aligned with `auto` or other style not specifying a static number?

Reply:
hmmm, Checking

Reply:
@leecalcote MenuItem is actually already styled with MUI internal styling, So auto may gonna have problems here but however, we can go for default styling also, no custom padding making our list little bigger but also looks good. 
Here--
![Screenshot from 2022-05-02 23-53-01](https://user-images.githubusercontent.com/56961488/166303398-1d7b331d-8ac1-43b7-9efd-854e219e199d.png)

Reply:
@leecalcote I am finding no way to style menuitem without specifying static number because here we have to nullify the first padding given by default in MUI, should I resolve this conversation?

Reply:
@Nikhil-Ladha any thoughts here? We can dismiss my comment if its not worth the time.

Reply:
Yeah, unfortunately, the way MUI is....... adding static values for additional classes is required.